sopel/modules/reddit.py

```diff
@@ -47,13 +47,17 @@ def shutdown(bot):
 
 @rule('.*%s.*' % post_url)
 def rpost_info(bot, trigger, match=None):
-    r = praw.Reddit(
-        user_agent=USER_AGENT,
-        client_id='6EiphT6SSQq7FQ',
-        client_secret=None,
-    )
     match = match or trigger
-    s = r.submission(id=match.group(2))
+    try:
+        r = praw.Reddit(
+            user_agent=USER_AGENT,
+            client_id='6EiphT6SSQq7FQ',
+            client_secret=None,
+        )
+        s = r.submission(id=match.group(2))
+    except Exception:
```

Comment:
I don't like this, but there's no practical way of catching the real exception because it's moved locations in `praw`. I'm not a fan of having a try/except tree of imports just for this, so I'll let it slide. :stuck_out_tongue: